### PR TITLE
Make Testing Tools Section

### PR DESCRIPTION
**Pull request by davidjbradshaw**
I though it might make sense to split out the test tooling from frameworks into it's own section

### CODE REVIEW

**Comment by davidjbradshaw** [comment]
Sorry I don't seem to be able to fix this for some reason

File: readme.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
   - [Misc](#misc)
   - [Practices](#practices)
   - [Style](#style)
+  - [Testing Tools](testing-tools)
```
Comment (dustinspecker) [inline readme.md]:
```suggestion
  - [Testing Tools](testing-tools)
  + [Testing Tools](#testing-tools)
```

I think that's what is needed by awesome-lint.

**Comment by dustinspecker** [comment]
I like this a lot. Good idea. Added a comment of what I think the issue is.

**Comment by davidjbradshaw** [comment]
That was dumb on my part, thanks have merged your suggestion and it works now

**Comment by dustinspecker** [comment]
Thanks!

**Pull request merged**